requirements.txt

```diff
@@ -3,6 +3,7 @@ coverage
 Flask
 Flask-DotEnv
 Flask-HTTPAuth
+Flask-JWT
 Flask-Login
 Flask-Migrate
 Flask-Script
```

Comment:
It's normally a good idea to have specific version numbers for your packages. Might prevent things from blowing 🔥 up when packages fail to play along.